terraform/securityhub.tf

```diff
@@ -24,3 +30,38 @@ resource "aws_securityhub_organization_admin_account" "default-region-administra
   ]
   admin_account_id = aws_organizations_account.organisation-security.id
 }
+
+################################
+# Member accounts in eu-west-2 #
+################################
+
+resource "aws_securityhub_member" "eu-west-2" {
+  # Note that this resource returns an UnprocessedAccount error when a member is removed, so you need to login to the AWS console
+  # and do it manually, however, no one should be removed once enrolled.
+  # See: https://docs.aws.amazon.com/guardduty/latest/APIReference/API_UnprocessedAccount.html
+  for_each = {
+    for account in local.enrolled_into_securityhub :
+    account.name => account.id
+  }
+  provider = aws.organisation-security-eu-west-2
+
+  # We want to add these accounts as members within the Organisation Security account
+  account_id = each.value
+  email      = "fake@email.com"
```

Comment:
`email.com` is a real live domain. For this sort of thing, you should always use an IANA-

Reply:
(Unless AWS blocks that, in which case, let's reserve `example.justice.gov.uk`.)

Reply:
Ooo, thanks @SteveMarshall 👍🏼